### PR TITLE
Update the README.markdown file in pdc-client repo

### PR DESCRIPTION
**Pull request by bliuredhat**
JIRA:PDC-1702

### CODE REVIEW

**Comment by simozhan** [comment]
:+1:

File: README.markdown [hunk 1]
```diff
@@ -11,7 +11,8 @@ You can obtain the client from the same repository where PDC server is.
 
 The client can read server connection details from a configuration file.
 The configuration file should be located in
-`/etc/pdc/client_config.json` or in `~/.config/pdc/client_config.json`.
+`/etc/pdc.d/` directory which contains `redhat.json`and `fedora.json` files,
```
Comment (simozhan) [inline README.markdown]: i think it is better not to mention redhat.json in github repo.

**Comment by ycheng-aa** [comment]
:+1:

**Comment by simozhan** [comment]
:+1:

**Pull request merged**